doglover/src/components/Breed.tsx

```diff
@@ -0,0 +1,42 @@
+import {useState, useEffect} from 'react';
+import { useParams } from 'react-router-dom';
+import {Heading, Image, Container,  Grid} from '@chakra-ui/react';
+
```

Comment:
Similarly with AllBreeds.tsx when rendering this component, "some" console errors exist.